pre_commit/clientlib.py

```diff
@@ -19,6 +20,14 @@ def check_language(v):
         )
 
 
+def check_type_tag(tag):
+    if tag not in ALL_TAGS:
+        raise schema.ValidationError(
+            'Type tag {!r} is not recognized.  '
+            'Try upgrading identify and pre-commit?'.format(tag),
+        )
```

Comment:
Yeah... the more common errors they're going to get is "no file hooks.yaml" and "missing key 'files'" though -- and unfortunately no way to retroactively improve error messages in old versions :S